setup.py

```diff
@@ -10,6 +10,8 @@
     install_requires=[
         "backoff",
         "celery>=5",
+        "gemlock_parser @ https://github.com/containerbuildsystem/gemlock-parser/archive"
+        "/a2f0f4020e07b1e87813a3254eb3e40047d8e981.zip",
```

Comment:
This can just be `gemlock_parser`. That's the difference between setup.py and requirements.txt: setup.py defines the dependencies you need (and their version ranges, optionally), requirements.txt defines the exact versions and how to get them.

Reply:
Oh, thanks